site/port5.html

```diff
@@ -0,0 +1,96 @@
+<html>
+
+	<link rel="stylesheet" href="style_port.css"
+
+	<head>
+		<title>The Digital Yogi</title>
+	</head>
+	
+	<body background="images/background pattern.png">
+		<nav>
+		  <a href="about2.html">About Me</a> |
+		  <i><a href="port5.html">Portfolio</a> |</i>
+		</nav>
+
+		<h1>GABRIELE GEIER</h1>
+
+		<hr> &nbsp;
+		
+			<h2>EXPERIENCE</h2>
+
+			<p>I currently work for the Lung Cancer Alliance, where I use my communications skills, including social media management, graphic design and brand marketing, to help those living with and at risk for lung cancer. With the skills I am developing in graphic design and web development, I am helping reimage a disease that is so oftn overlooked and misunderstood. </p>
+			&nbsp;
+
+				<h3>Portfolio</h3>
+				<p>
+					<ul>	
+						<li><span class="title">Web Developer</span>, <span class="tool">Relaxr</span> <span class="time">December 2015-Present</span></li>
+							<p> Developed a multi-column layout blog, landing page and contact forms that render on mobile devices.</p>
+							<img src="images/relaxr_deliverable.png" border=1 height= 400 align= left>
+					</ul>
+				</p>
+					<br> </br>
```

Comment:
I see what you mean by using all of the ```<br>``` tags to add space.  We did not go over margin in time for this homework assignment, but for next time, think about how you can add a margin to your ```<p>``` tags to create this space instead.